spec/spec_helper.rb

```diff
@@ -7,14 +7,16 @@
 
 bootfile = File.expand_path(File.dirname(__FILE__) + '/../config/boot')
 require bootfile
+require 'ruby-vips'
+require 'byebug'
```

Comment:
please add byebug to the gemspec if you want to leave this line in.  Better yet, add `pry-byebug` instead.

Reply:
byebug is already in the gemspec file in this PR, happy to swtich to pry-byebug (though unclear what the difference is)

Reply:
pry-byebug adds some capabilities to byebug, such as being able to do `ls object` to get all the methods on `object` and where they are coming from.  It also has color output for easier scanning of code.  But it's a personal preference.  I believe it's a wrapper around byebug so you can still use byebug (and the commands to step through the code are the same, I believe)

It's ... er ... awesome! ... that I didn't see you added byebug to the gemspec in this PR.  Sorry.

Reply:
ok -- changing to `pry-byebug` in next commit (pending)